src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -103,7 +103,27 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
     tezosConf.network
   )
 
-  try {mainLoop(0)} finally {shutdown()}
+  /**
+    * Tries to fetch blocks head to verify if connection with Tezos node was successfully established
+    */
+  @tailrec
+  def checkConnection(): Unit = {
```

Comment:
Can we give this a more specific name like `checkTezosConnection`?

Reply:
changed